GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeView+NumberView.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+extension CallVisualizer.VisitorCodeView {
+    class NumberView: UILabel {
+        struct Props: Equatable {
+            let character: Character
+        }
+
+        override init(frame: CGRect) {
+            super.init(frame: frame)
+            self.font = .boldSystemFont(ofSize: 40)
+            self.numberOfLines = 0
+            self.textColor = UIColor.black // style.messageColor
```

Comment:
should it be uncommented?

Reply:
You're right. Fixed. [676c74c](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/676c74ceb781899d4ca4fa8a8ab684c8a1837308)